.github/workflows/e2e_additional_editor_and_player.yml

```diff
@@ -31,15 +31,18 @@ jobs:
       - name: Install chrome
         if: ${{ steps.check-risk.outputs.is-low-risk != 0 }}
         run: python -m scripts.install_chrome_for_ci
+      - name: Install ffmpeg
+        if: ${{ steps.check-risk.outputs.is-low-risk != 0 }}
+        run: sudo apt install ffmpeg
       - name: Run Additional Editor E2E Test
         if: ${{ steps.check-risk.outputs.is-low-risk != 0 }}
-        run: xvfb-run -a python -m scripts.run_e2e_tests --suite="additionalEditorFeatures" --prod_env
+        run: xvfb-run -a --server-args="-screen 0, 1280x1024x24" python -m scripts.run_e2e_tests --suite="additionalEditorFeatures" --prod_env
```

Comment:
The tests try and set a screen size of 1285x1000, so maybe we should make the display at least that big

Reply:
changed the screen size to 1285x1000 for all suites